miner/worker.go

```diff
@@ -1075,35 +1111,54 @@ func (w *worker) fillTransactions(interrupt *int32, env *environment) {
 			localTxs[account] = txs
 		}
 	}
+	lenLocals, lenNewLocals, localEnvTCount, lenRemotes, lenNewRemotes, remoteEnvTCount := 0, 0, 0, 0, 0, 0
 	if len(localTxs) > 0 {
+		lenLocals = len(localTxs)
 		txs := types.NewTransactionsByPriceAndNonce(env.signer, localTxs, env.header.BaseFee)
+		lenNewLocals = txs.GetTxs()
 		if w.commitTransactions(env, txs, interrupt) {
 			return
 		}
+		localEnvTCount = env.tcount
 	}
 	if len(remoteTxs) > 0 {
+		lenRemotes = len(remoteTxs)
 		txs := types.NewTransactionsByPriceAndNonce(env.signer, remoteTxs, env.header.BaseFee)
+		lenNewRemotes = txs.GetTxs()
 		if w.commitTransactions(env, txs, interrupt) {
 			return
 		}
+		remoteEnvTCount = env.tcount
 	}
+	fillTransactionsSpan.SetAttributes(
```

Comment:
This won't be executing if you return from the function before reaching this point